README_libf3d.md

```diff
@@ -0,0 +1,202 @@
+# libf3d - A library to render 3D meshes
+
+By Michael Migliore and Mathieu Westphal.
+
+libf3d is a BSD-licensed C++ library to open and render 3D meshes. It is of course used by F3D.
+libf3d API is simple and easy to learn. Python bindings are provided through pybind11.
+
+# Getting Started
+
+Rendering a file and starting the interaction is very easy:
+
+```
+// Create a f3d::engine
+f3d::engine eng();
+
+/ Add a file to the loader and load it
+eng.getLoader().addFile("path/to/file.ext").loadFile();
+
+// Render the file in the window
+eng.getWindow().render();
```

Comment:
is it required? Starting the interactor triggers a render, right?

Reply:
If you want to showcase the access to the window, then maybe call `setSize` or something

Reply:
indeed, should not be necessary, but i forgot to add a test, I will.

Reply:
It is tested exstensivedly by F3D.